Util/ai_command.py

```diff
@@ -3,12 +3,58 @@
 from collections import namedtuple
 
 
-class AICommand(namedtuple('AICommand', 'robot_id target kick_type kick_force charge_kick dribbler_active path '
-                                        'cruise_speed end_speed ball_collision')):
 
+class AICommand(namedtuple('AICommand', 'target kick_type kick_force charge_kick dribbler_active '
+                                        'cruise_speed end_speed ball_collision pathfinder_on')):
     __slots__ = ()
 
-    def __new__(cls, robot_id, target=None, kick_type=None, kick_force=0, charge_kick=False, dribbler_active=False,
-                path=None, cruise_speed=1, end_speed=0, ball_collision=True):
-        return super().__new__(cls, robot_id, target, kick_type, kick_force, charge_kick, dribbler_active, path,
-                               cruise_speed, end_speed, ball_collision)
+
+class CmdBuilder:
+
+    def __init__(self):
+        # Does are the defaults values of AICommand
+        # A robot should stay idle with those value
+        self.target = None
+        self.kick_type = None
+        self.kick_force = 0
+        self.charge_kick = False
+        self.dribbler_active = False
+        self.cruise_speed = 0
+        self.end_speed = 0
+        self.ball_collision = True
+        self.pathfinder_on = True
+
+    def withMoveTo(self, target, cruise_speed=1, end_speed=0, ball_collision=True):
```

Comment:
could be a @classmethod.

`@classmethod
def withMoveTo(cls, ...)
    obj = cls()
    bog.arg = arg
    return cls(args)
`